internal/pkg/deploy/cloudformation/stack/transformers_test.go

```diff
@@ -34,18 +34,12 @@ func Test_convertSidecar(t *testing.T) {
 		wanted    *template.SidecarOpts
 		wantedErr error
 	}{
-		"invalid port": {
-			inPort: aws.String("b/a/d/P/o/r/t"),
-
-			wantedErr: fmt.Errorf("cannot parse port mapping from b/a/d/P/o/r/t"),
-		},
 		"good port without protocol": {
 			inPort:      aws.String("2000"),
 			inEssential: true,
 
 			wanted: &template.SidecarOpts{
 				Name:       "foo",
-				Port:       aws.String("2000"),
```

Comment:
It doesn't sound right to me that with `inPort:"2000"`, the expected outcome does not have any `PortMappings` 🤔. I know this is because the exposed ports are passed in as `nil` in the unit test, but I think we shouldn't leave the unit tests like this, that the `wanted` outcome looks wrong comparing with `in`.

I feel like we should either:
1. Remove `inPort` from these test cases. That is, we don't test at all if a sidecar's port is converted as expected in the unit test. We instead rely on integration test to make sure it's all right.
2. Or, pass in a mock `[]manifest.ExposedPort` in `got, err := convertSidecars(sidecar, mockExposedPorts)` at the current L225.

Personally, I am fine either way!